### PR TITLE
FORMS-8659 Include WKND and Easel theme artifact in archetype

### PR DESCRIPTION
**Pull request by suryansh29**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/main/archetype/ui.apps/pom.xml [hunk 1]
```diff
@@ -55,6 +55,7 @@
                     <group>${groupId}</group>
                     <name>${artifactId}</name>
                     <packageType>application</packageType>
+                    <skipSubPackageValidation>true</skipSubPackageValidation>
```
Comment (barshat7) [inline src/main/archetype/ui.apps/pom.xml:59]: why are we skipping package validation?

Comment (suryansh29) [reply src/main/archetype/ui.apps/pom.xml:59]: We have wknd and easel themes in the hierarchy which are zips. This plugin thinks that they are packages and fail the validation. skipSubPackageValidation solves that.
I tried other methods which can possibly skip validation for our theme zips but no success.

File: src/main/archetype/ui.apps/src/main/content/META-INF/vault/filter.xml [hunk 2]
```diff
@@ -3,6 +3,10 @@
     <filter root="/apps/${appId}/clientlibs"/>
     <filter root="/apps/${appId}/components"/>
     <filter root="/apps/${appId}/i18n"/>
+#if ( $includeForms == "y" || $includeFormsenrollment == "y" || $includeFormscommunications == "y" || $includeFormsheadless == "y")
+    <filter root="/apps/fd/af/themes"/>
+    <filter root="/apps/fd/af/theme-clientlibs"/>
+#end
```
Comment (pankaj-parashar) [inline src/main/archetype/ui.apps/src/main/content/META-INF/vault/filter.xml:11]: both filter should have conditional inclusion based on aem version.

**Review by pankaj-parashar — COMMENTED** [review]
@suryansh29, have we added latest fixes from WKND and Easel theme repos in this PR?

File: src/main/archetype/ui.apps/src/main/content/META-INF/vault/filter.xml (hunk 2 above)
Comment (suryansh29) [reply src/main/archetype/ui.apps/src/main/content/META-INF/vault/filter.xml:11]: In future we do plan to add theme client libs incase a user migrates from 65 to cloud, didn't add explicit check for that

Comment (pankaj-parashar) [reply src/main/archetype/ui.apps/src/main/content/META-INF/vault/filter.xml:11]: Let's make it conditional for now. When we support it in the future, we will remove the condition.

**Comment by pankaj-parashar** [comment]
@suryansh29 , Please update the PR title to reflect theme name properly - WKND and Easel.

File: src/main/archetype/ui.apps/pom.xml (hunk 1 above)
Comment (barshat7) [reply src/main/archetype/ui.apps/pom.xml:59]: can we put this under the condition only if forms is included? 
#if ( $includeForms == "y" || $includeFormsenrollment == "y" || $includeFormscommunications == "y" || $includeFormsheadless == "y

Comment (suryansh29) [reply src/main/archetype/ui.apps/pom.xml:59]: done

File: src/main/archetype/ui.apps/src/main/content/META-INF/vault/filter.xml (hunk 2 above)
Comment (suryansh29) [reply src/main/archetype/ui.apps/src/main/content/META-INF/vault/filter.xml:11]: done

**Review by barshat7 — APPROVED** [review]

**Pull request merged**